content/en/_index.md

```diff
@@ -197,6 +236,50 @@ public static void listInfoTypes() throws IOException {
   }
 }
 {{< /tab >}}
+{{< tab header="Go" >}}
+import (
+	"context"
+	"fmt"
+
+	dlp "cloud.google.com/go/dlp/apiv2"
+	dlppb "google.golang.org/genproto/googleapis/privacy/dlp/v2"
+)
+
+// listInfoTypes lists the types of sensitive information the DLP API supports.
```

Comment:
Should this have the `main()` function here?

Reply:
I've matched the Java usage above. I think using a non-main function makes it more copy-paste friendly.